e2e/test/scenarios/admin/i18n/content-translation/dashboards.cy.spec.ts

```diff
@@ -85,8 +85,9 @@ describe("scenarios > content translation > static embedding > dashboards", () =
 
         H.saveDashboard();
 
-        H.openStaticEmbeddingModal({
-          acceptTerms: false,
+        H.openLegacyStaticEmbeddingModal({
```

Comment:
I don't understand if this is testing the guest embed or the old static embed. Based on the test name it's testing Guest embeds, but this function should open the legacy static embed modal.

Reply:
it's for legacy